manifests/server/startup.pp

```diff
@@ -0,0 +1,90 @@
+# == Define: dns::server::startup
+#
+# BIND server template-based configuration definition.
+#
+# === Parameters
+#
+# $startup_file
+# Must be an absolute path to the bind9 file as title
+#
+# [*resolvconf*]
+#   Set string "yes" or "no". Default: no
+#
+# [*options*]
+#   String to set user and IPv4/IPv& Support. Default: -u bind
+#
+# [*rootdir*]
+#   Must be an absolute path to set up the chroot environment
+#
+# [*enable_zone_write*]
+#   If SELinux is disabled, then allow named to write its zone files
+#   Set string "yes", "no" or leave empty. Default: empty
+#
+# [*enable_sdb]
+#   Enables use of 'named_sdb', which has support for the ldap
+#   Set string "yes", "no" or leave empty. Default: empty
+#
+# [*keytab_file*]
+#   Must be an absolute path to set up the keytab file
+#
+# [*disable_zone_checking*]
+#   If you set this option to 'yes' then initscript doesn't perform named-checkzone
+#   Set string "yes", "no" or leave empty. Default: empty
+#
+# === Examples
+#
+#  dns::server::startup { '/etc/default/bind9':
```

Comment:
I don't want to make the same mistake as the options file. This shouldn't be a defined type that users have to "remember" to add to their manifest. init.pp should include this.